packages/runtime/container-runtime/src/dataStoreContexts.ts

```diff
@@ -68,13 +68,15 @@ import { FluidDataStoreContext, LocalFluidDataStoreContext } from "./dataStoreCo
     }
 
     /**
-     * Prepare and return the unbound context with the given id so it can be bound.
+     * Return the unbound local context with the given id,
+     * or undefined if it's not found or not unbound.
      */
-    public prepContextForBind(id: string): LocalFluidDataStoreContext {
-        assert(this.notBoundContexts.has(id), "Store being bound should be in not bounded set");
-        assert(this._contexts.has(id), "Attempting to bind to a context that hasn't been added yet");
+    public getUnbound(id: string): LocalFluidDataStoreContext | undefined {
+        const context = this._contexts.get(id);
+        if (!context || !this.notBoundContexts.has(id)) {
```

Comment:
```suggestion
        if (context === undefined || !this.notBoundContexts.has(id)) {
```
thought we had an eslint rule against truthy checks. might be off for this project